Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -143,52 +170,36 @@ private void SetControls()
             }
         }
 
-        public void RefreshALMSolutionSettings()
+        private void ChangeALMType()
         {
-            if ((ALMIntegration.eALMType) WorkSpace.Instance.Solution.AlmType == ALMIntegration.eALMType.QC && QCRadioButton.IsChecked == false)
+            if (CurrentAlmConfigurations.AlmType == GingerCoreNET.ALMLib.ALMIntegration.eALMType.QC && QCRadioButton.IsChecked != null ? (bool)QCRadioButton.IsChecked : false)
             {
                 QCRadioButton.IsChecked = true;
-                PasswordTextBox.Password = "";
-                UserNameTextBox.Text = "";
-                StyleRadioButtons();
             }
-            else if ((ALMIntegration.eALMType) WorkSpace.Instance.Solution.AlmType == ALMIntegration.eALMType.RQM && RQMRadioButton.IsChecked == false)
+            else if (CurrentAlmConfigurations.AlmType == GingerCoreNET.ALMLib.ALMIntegration.eALMType.RQM && RQMRadioButton.IsChecked != null ? (bool)RQMRadioButton.IsChecked : false)
             {
                 RQMRadioButton.IsChecked = true;
-                PasswordTextBox.Password = "";
-                UserNameTextBox.Text = "";
-                StyleRadioButtons();
             }
-            else if ((ALMIntegration.eALMType) WorkSpace.Instance.Solution.AlmType == ALMIntegration.eALMType.RALLY && RallyRadioButton.IsChecked == false)
+            else if (CurrentAlmConfigurations.AlmType == GingerCoreNET.ALMLib.ALMIntegration.eALMType.Jira && JiraRadioButton.IsChecked != null ? (bool)JiraRadioButton.IsChecked : false)
+            {
+                JiraRadioButton.IsChecked = true;
+            }
+            else if (CurrentAlmConfigurations.AlmType == GingerCoreNET.ALMLib.ALMIntegration.eALMType.RALLY && RallyRadioButton.IsChecked != null ? (bool)RallyRadioButton.IsChecked : false)
             {
                 RallyRadioButton.IsChecked = true;
-                PasswordTextBox.Password = "";
-                UserNameTextBox.Text = "";
-                StyleRadioButtons();
             }
-        }
-
-        private void ClearALMConfigs()
-        {
-            ServerURLTextBox.Text = null;
-            UserNameTextBox.Text = null;
-            PasswordTextBox.Password = null;
-             WorkSpace.Instance.UserProfile.ALMPassword = null;
-            DomainComboBox.SelectedItem = null;
-            DomainComboBox.SelectedValue = null;
-            DomainComboBox.Items.Clear();
-            ProjectComboBox.SelectedItem = null;
-            ProjectComboBox.SelectedValue = null;
-            ProjectComboBox.Items.Clear();
-            RestAPICheckBox.IsChecked = false;
+            else if (CurrentAlmConfigurations.AlmType == GingerCoreNET.ALMLib.ALMIntegration.eALMType.Qtest && qTestRadioButton.IsChecked != null ? (bool)qTestRadioButton.IsChecked : false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)